applications/ShapeOptimizationApplication/python_scripts/analyzer_internal.py

```diff
@@ -118,8 +122,12 @@ def __CreateResponseFunctions( specified_responses, model ):
             "mesh_based_packaging",
             "surface_normal_shape_change",
             "face_angle",
+            "angle_of_attack",

```

Comment:
i would add `airfoil` or `2D`to these names, since the implementation is specific to this kind of geometries.